yogstation/code/modules/admin/sql_message_system.dm

```diff
@@ -370,8 +371,8 @@
 						alpha = 10
 						skipped = TRUE
 					alphatext = "filter: alpha(opacity=[alpha]); opacity: [alpha/100];"
-
-			var/list/data = list("<p style='margin:0px;[alphatext]'> <b>[timestamp] | [server] | [admin_key]")
+			var/list/data = list("<p style='margin:0px;[alphatext]'> <b>[timestamp] | [server] | [admin_key] | ")
+			data += "Round: #<a href='?_src_=holder;[HrefToken()];viewdemo=[roundnumber]'>[roundnumber]</a>"
```

Comment:
Extra thing, find the first stored replays manually and check that the round number >= to that so people don't attempt to view replays from pre-replay days